news/2020-19-11-mappers-report-september-october.md

```diff
@@ -0,0 +1,211 @@
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+The cycle of life never ends. And the same goes for the cycle of Beatmap Nominators! As every month we show you who joined the team and who left the team!
+
+### BN Additions
+
+- **osu!:** [StarCastler](https://osu.ppy.sh/users/12402453), [Luscent](https://osu.ppy.sh/users/2688581), [Beomsan](https://osu.ppy.sh/users/3626063), [Petal](https://osu.ppy.sh/users/7354729), [Blitzifyyy](https://osu.ppy.sh/users/11787713), [Astronic](https://osu.ppy.sh/users/9320502), [Akito](https://osu.ppy.sh/users/5716327), [Sylvarus](https://osu.ppy.sh/users/4505918), [riffy](https://osu.ppy.sh/users/597957) & [Krisom](https://osu.ppy.sh/users/99269)
+- **osu!taiko:** [Cynplytholowazy](https://osu.ppy.sh/users/3901754) & [Jerry](https://osu.ppy.sh/users/605973)
+- **osu!catch:** [Nelly](https://osu.ppy.sh/users/4741164), [Spectator](https://osu.ppy.sh/users/702598) & [Phob](https://osu.ppy.sh/users/6069462)
+- **osu!mania:** [Lirai](https://osu.ppy.sh/users/3734394), [Fresh Chicken](https://osu.ppy.sh/users/3984370) & [Unpredictable](https://osu.ppy.sh/users/7560872)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open.
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+### BN Departures
+
+- **osu!:** [Venix](https://osu.ppy.sh/users/5999631), [Trynna](https://osu.ppy.sh/users/2652951), [[Karcher] 2](https://osu.ppy.sh/users/9892196), [NeKroMan4ik](https://osu.ppy.sh/users/11387664), [Ryuusei Aika](https://osu.ppy.sh/users/7777875), [Fall](https://osu.ppy.sh/users/4800816) & [Nikakis](https://osu.ppy.sh/users/4351739)
+- **osu!taiko:** [Roger](https://osu.ppy.sh/users/1538223),  [tasuke912](https://osu.ppy.sh/users/2774767), [Ulqui](https://osu.ppy.sh/users/1263669) & [incandescence](https://osu.ppy.sh/users/6256027)
+- **osu!catch:** [wonjae](https://osu.ppy.sh/users/5032045), [Sinnoh](https://osu.ppy.sh/users/4236057), [Bunnrei](https://osu.ppy.sh/users/829284) & [Baron](https://osu.ppy.sh/users/10286499)
+- **osu!mania:** [Keiga](https://osu.ppy.sh/users/6866022), [Garalulu](https://osu.ppy.sh/users/757783),  [Kamuy](https://osu.ppy.sh/users/7439226),  [Muse Dash](https://osu.ppy.sh/users/13695676) & [Kawawa](https://osu.ppy.sh/users/4647754)
+
+### NAT Additions
+
+A star, shining brighter than the sun has ascended from the Beatmap Nominators and is shining now in a scary red color on the osu! sky, [hypercyte](https://osu.ppy.sh/users/9155377) is the newest addition to the Nomination Assessment Team! After two years of work in the Beatmap Nominators, he will now support the osu! NAT with the evaluation of applications and Beatmap Nominators, to ensure a constant flow of new additions to the BNG.
+So once more: Welcome hypercyte!
```

Comment:
iirc, having the lines like this will put them into a single paragraph. If you meant to have them be two separate lines, then it needs to be either:
```suggestion
A star, shining brighter than the sun has ascended from the Beatmap Nominators and is shining now in a scary red color on the osu! sky, [hypercyte](https://osu.ppy.sh/users/9155377) is the newest addition to the Nomination Assessment Team! After two years of work in the Beatmap Nominators, he will now support the osu! NAT with the evaluation of applications and Beatmap Nominators, to ensure a constant flow of new additions to the BNG.

So once more: Welcome hypercyte!
```

Reply:
or
```suggestion
A star, shining brighter than the sun has ascended from the Beatmap Nominators and is shining now in a scary red color on the osu! sky, [hypercyte](https://osu.ppy.sh/users/9155377) is the newest addition to the Nomination Assessment Team! After two years of work in the Beatmap Nominators, he will now support the osu! NAT with the evaluation of applications and Beatmap Nominators, to ensure a constant flow of new additions to the BNG.\
So once more: Welcome hypercyte!
```

Reply:
otherwise, just put them in as a single paragraph
```suggestion
A star, shining brighter than the sun has ascended from the Beatmap Nominators and is shining now in a scary red color on the osu! sky, [hypercyte](https://osu.ppy.sh/users/9155377) is the newest addition to the Nomination Assessment Team! After two years of work in the Beatmap Nominators, he will now support the osu! NAT with the evaluation of applications and Beatmap Nominators, to ensure a constant flow of new additions to the BNG. So once more: Welcome hypercyte!
```